### PR TITLE
Update User Guide

### LINKED ISSUE #38: Update User Guide
Add features to be implemented in v1.2

### PR DESCRIPTION
**Pull request by xz0127**
Update User Guide by adding 'clear', 'help', 'exit' command.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -60,12 +60,14 @@ Nuudle is a **desktop app for managing patient records and clinic appointments,
 
 ### Viewing help : `help`
 
-Shows a message explaining how to access the help page.
+Shows a message explaining how to access Nuudle's help page.
```
Comment (WadeForward) [inline docs/UserGuide.md]: would `help page of Nuudle` be better? or just another way to avoid not using `'s`

**Review by JinHao-L — COMMENTED** [review]
I'm not sure but I think that the `example` for a single-word command can be removed. What do you think?

Other than this point, the rest LGTM 👍

**Review by WadeForward — COMMENTED** [review]
LGTM! 👍

**Review by YangYue128-helen — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -166,15 +165,15 @@ Format `cancel APPT_ID`
 Example:
 * `cancel 202003081000` deletes the appointment happening on 08/03/2020 10am.
 
-### Clearing all entries : `clear`
+### Clearing all patient entries : `clear`
```
Comment (JinHao-L) [inline docs/UserGuide.md]: Seeing that appointment is dependent on the patient, should it be clear all appointment entries instead? What do you think?

Comment (xz0127) [reply docs/UserGuide.md]: Makes sense, I will make the change. Thanks! 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (xz0127) [reply docs/UserGuide.md]: Ok, will make the change! Thanks! 👍

**Review by WadeForward — APPROVED** [review]
Brilliant!

**Pull request merged**